src/c_api/c_api.cc

```diff
@@ -162,6 +195,24 @@ void CustomFComputeDispatcher(const std::string op_name,
     return workspace.dptr_;
   };
 
+  // create lambda that allocates memory for sparse and updates MXSparse.
```

Comment:
can we change the comment to:
  // create lambda that allocates memory for sparse and returns allocated arrays for data, indices and indptr